app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
```

Comment:
I was talking about this with @pcraig3.

The first point is that there's not really... anything... to fetch from the content loader for this story. The human-readable lot names come from the `get_framework` call and the status strings are `Live` and `Closed`. So I thought dragging pointlessly tiny things from the content loader would get me slapped with the "overkill" label.

On the other hand, the gcloud search method doesn't _really_ have a way of dealing with fields etc. that _aren't_ sourced from the content loader. So I would have had to extend the gcloud mechanisms to cope with these - however, if we're possibly going to take the next step in filtering these and throw the briefs into the search api along with other filterable parameters, it would then make that addition redundant. This (the way I've done it), well you _could_ just chuck this away at that point if you decided.

Thirdly, if we're going to choose one approach to be able to "extend" to cover all cases, from a maintainability point of view I would *strongly* argue in favour of basing it on top of a proven, object-oriented, heavily tested & highly documented forms implementation like wtforms rather than a home-baked solution which is not really object-oriented and already a little hard for a newbie to decode. I'm confident wtforms would be able to build its fields from the content loader if necessary.

And the lovely thing about object-oriented forms implementations like wtforms is they have extensibility in their very nature - you can use inheritance to plug in & out bits of functionality as you like. You have to get you hands a lot dirtier with procedural code...